client/js/app/components/explorer/query_builder/extraction_properties_filter.js

```diff
@@ -0,0 +1,45 @@
+var React = require('react');
```

Comment:
For completely new files, I think we should opt to use es2017+ syntax. This includes things like `let` and other modern language features. This is up for debate of course, but I think if we don't start moving our codebase towards that now we'll never get there without a full re-write. What's your opinion here?